src/app/workspace/page.tsx

```
@@ -43,9 +51,38 @@ const WorkspacePage = () => {
-			<p className="text-sm">Loading your workspace...</p>
+		<div className="bg-primary flex h-full flex-1 flex-col items-center justify-center gap-4 text-white p-8">
+			<Loader className="size-8 animate-spin mb-4" />
+			<p className="text-xl font-bold">Loading your workspace...</p>
+			
+			<div className="mt-8 flex flex-col items-center gap-4">
+				<p className="text-sm text-white/70">Taking too long?</p>
+				<div className="flex gap-4">
+					<button 
+						onClick={() => setOpen(true)}
+						className="px-4 py-2 bg-white text-primary rounded-md font-medium text-sm hover:bg-white/90"
+					>
```

Biome (2.4.13)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)